lms/templates/calculator/toggle_calculator.html

```diff
@@ -1,144 +1,158 @@
-                    <td>${_("Percent sign (%) and metric affixes (d, c, m, u, n, p, k, M, G, T)")}</td>
-                    <td>20%<br />
-                      20c<br />
-                      418T
-                    </td>
-                  </tr>
-                  <tr>
-                    <th scope="row">${_("Basic functions")}</th>
-                    <td>abs, exp, fact or factorial, ln, log2, log10, sqrt</td>
-                    <td>abs(x+y)<br />
-                      sqrt(x^2-y)
-                    </td>
-                  </tr>
-                  <tr>
-                    <th scope="row">${_("Trigonometric functions")}</th>
-                    <td>sin, cos, tan, sec, csc, cot<br />
-                      arcsin, sinh, arcsinh, etc.<br />
-                    </td>
-                    <td>sin(4x+y)<br />
-                      arccsch(4x+y)
-                    </td>
-                    <td></td>
-                  </tr>
-                  <tr>
-                    ## Translators: Please see http://en.wikipedia.org/wiki/Scientific_notation
-                    <th scope="row">${_("Scientific notation")}</th>
-                    ## Translators: 10^ is a mathematical symbol. Please do not translate.
-                    <td>${_("10^ and the exponent")}</td>
-                    <td>10^-9</td>
-                  </tr>
-                  <tr>
-                    ## Translators: this is part of scientific notation. Please see http://en.wikipedia.org/wiki/Scientific_notation#E_notation
-                    <th scope="row">${_("e notation")}</th>
-                    ## Translators: 1e is a mathematical symbol. Please do not translate.
-                    <td>${_("1e and the exponent")}</td>
-                    <td>1e-9</td>
-                  </tr>
-                </tbody>
-              </table>
-            </li>
-          </ul>
-        </div>
-      </div>
+    <button title="${_('Open Calculator')}" aria-controls="calculator_wrapper" aria-expanded="false" class="calc utility-control-button">
+        <span class="icon fa fa-calculator" aria-hidden="true"></span>
+    </button>
+    
+    <div id="calculator_wrapper">
+        <form id="calculator">
+            <div class="input-wrapper">
+                <label for="calculator_input" class="label-calc-input sr">${_('Enter equation')}</label>
+                <input type="text" id="calculator_input" title="${_('Calculator Input Field')}" aria-describedby="calculator_output" />
+                
+                <div class="help-wrapper">
+                    <p class="sr" id="hint-instructions">${_('Use the arrow keys to navigate the tips or use the tab key to return to the calculator')}</p>
+                    
+                    <button type="button" title="${_('Hints')}" id="calculator_hint" aria-haspopup="true" aria-expanded="false" aria-controls="calculator_input_help" aria-describedby="hint-instructions"></button>
```

Comment:
Hey @clrux this is unrelated to your PR, but Screen reader users will encounter this text before they move focus to the button (that references it via aria-describedby).  This really only affects VO users, who will encounter everything in the DOM, and will occasionally affect Windows screen reader users when they are in read mode, but I still think we should address it.  Instead of help text on the button, why don't we make this screen reader text the first text in the help modal.  Perhaps put it on line 28 below.